docs/root/operations/admin.rst

```diff
@@ -158,6 +158,10 @@ modify different aspects of the server:
   Enable/disable different logging levels on different subcomponents. Generally only used during
   development.
 
+.. http:post:: /memory
+
+  Prints current memory allocation / heap usage, in bytes. Useful in lieu of printing all `/stats` and filtering to get the memory-related statistics.
```

Comment:
I don't have a problem with this, but I think it would be not too hard, and pretty useful, to give stats a filtered view -- the filter being applied during map construction, which would have the same benefit but might be more generally . useful, and wouldn't involve adding a new API and proto which would need to live forever.

This is really a matter of taste, though, and I'm fine with this approach if others are.

Reply:
Agreed, having a filter for a single stat would be useful for stat inspection. But the contents of `Memory::Stats` aren't _really_ statistics, they're just bundled in there at calltime from `MallocExtension`. So this seemed like a more direct way to get this bit of information.